src/Compilers/CSharp/Portable/BoundTree/Expression.cs

```diff
@@ -495,6 +495,23 @@ public override void Accept(OperationVisitor visitor)
         }
     }
 
+    internal partial class BoundTupleExpression 
+    {
+        //PROTOTYPE: we need to make this its own OperationKind 
```

Comment:
Please do it with `PROTOTYPE(tuples)`. We are likely to merge multiple features before they are all done (e.g. tuples and pattern-matching).